apps/myaccount/src/components/federated-associations/federated-associations.tsx

```diff
@@ -84,6 +97,17 @@ export const FederatedAssociations: FunctionComponent<FederatedAssociationsProps
         getFederatedAssociationsList();
     }, []);
 
+    /**
+     *
+     */
```

Comment:
Can we add a comment here or remove this comment block?

Reply:
A comment is added now.